cypress/e2e/regression/remove_owner.cy.js

```diff
@@ -25,7 +25,8 @@ describe('Remove Owners tests', () => {
     owner.verifyRemoveBtnIsEnabled().should('have.length', 2)
   })
 
-  it('Verify Tooltip displays correct message for Non-Owner', () => {
+  //TODO: Clarify correct behaviour for non-owner button visibility
+  it.skip('Verify remove owner button is disabled for Non-Owner', () => {
```

Comment:
what an issue do we have currently with the case? Not possible to disconnect signer?

Reply:
This is now updated as discussed.